src/Bundle/Config/GridConfigInterface.php

```diff
@@ -32,6 +32,8 @@ public function addGrid(GridBuilderInterface $gridBuilder): self;
 
         /**
          * Gets all configuration represented as an array.
+         *
+         * @return array<string, mixed>
```

Comment:
Is it necessary to provide more detailed shape since `grids` key is always present in current implementation? (<string, <int, mixed>> or something)